pegasus/sites.v3/code.org/views/google_analytics.js.erb

```diff
@@ -1,3 +1,14 @@
+window.dataLayer = window.dataLayer || [];
+function gtag(){dataLayer.push(arguments);}
+
+// Setup Google Analytics tag and dimensions
```

Comment:
Can we expand the comments to differentiate between the GA4 vs. GA3 parts?

Reply:
Yes!